pkg/vat.go

```diff
@@ -187,3 +190,14 @@ func (m metricsReporter) StreamClosed(id protocol.ID) {
 		m.Decr("rpc.connected")
 	}
 }
+
+type StreamErrorReporter struct {
+	l log.Logger
+	s network.Stream
+}
+
+func (r StreamErrorReporter) ReportError(err error) {
+	if r.l != nil {
+		r.l.WithField("stream", r.s.ID()).Warn(err)
```

Comment:
Oh, also, let's make this `DEBUG`.  Last time I checked this could be a bit noisy with QUIC.